proto/messages.proto

```diff
@@ -192,6 +192,7 @@ message Asset {
     optional string asset_id = 3;
```

Comment:
I would add a comment to "sha256" that says that this is obsolete but it stays required for backward compatibility. Or is there a way to solve this otherwise, @bennyn ?